packages/webdriverio/src/utils/interception/utils.ts

```diff
@@ -101,3 +105,84 @@ export function getPatternParam (pattern: URLPattern, key: keyof Omit<remote.Net
 
     return pattern[key].replaceAll('*', '\\*')
 }
+
+/**
+ * Converts a glob pattern to a URLPattern compatible regular expression pattern
+ * @param globPattern - The glob pattern to convert
+ * @returns A URLPattern instance representing the converted glob pattern
+ */
+export function globToURLPattern(globPattern: string): URLPattern {
+    // Helper function to convert a glob segment to regex
+    function convertGlobToRegex(glob: string): string {
+        // Step 1: Escape characters that have special meaning in regex
+        let regex = glob.replace(/[.+?^${}()|[\]\\]/g, '\\$&')
+        // Step 2: Convert glob patterns to regex patterns
+        // Convert ** (matches any number of directories/segments)
+        regex = regex.replace(/\\\*\\\*/g, '.*')
+
+        // Convert * (matches anything within a single segment)
+        regex = regex.replace(/\\\*/g, '[^/]*')
```

Comment:
The previous line changes ** to * so this will match that which is not what you want by the looks of it